processor/transformprocessor/internal/traces/processor.go

```diff
@@ -21,24 +21,24 @@ import (
 	"go.opentelemetry.io/collector/pdata/ptrace"
 	"go.uber.org/zap"
 
-	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/telemetryquerylanguage/contexts/tqltraces"
-	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/telemetryquerylanguage/tql"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/oteltransformationlanguage/contexts/ottltraces"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/oteltransformationlanguage/ottl"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/transformprocessor/internal/common"
 )
 
 type Processor struct {
-	queries []tql.Query
+	queries []ottl.Query
 	logger  *zap.Logger
 }
 
 func NewProcessor(statements []string, functions map[string]interface{}, settings component.ProcessorCreateSettings) (*Processor, error) {
-	tqlp := tql.NewParser(
+	ottlp := ottl.NewParser(
 		functions,
-		tqltraces.ParsePath,
-		tqltraces.ParseEnum,
-		common.NewTQLLogger(settings.Logger),
+		ottltraces.ParsePath,
+		ottltraces.ParseEnum,
+		common.NewOTTLLogger(settings.Logger),
 	)
-	queries, err := tqlp.ParseQueries(statements)
+	queries, err := ottlp.ParseQueries(statements)
```

Comment:
Likely outside the scope of this PR, but should the `Query` type be renamed now that it is no longer the TQL? I think `Statement` would work well, but there may be better alternatives.

Reply:
Yes it should be changed. It'll do that in a follow up PR